citesphere/src/main/resources/config.properties

```diff
@@ -40,12 +40,16 @@ _viaf_authority_search_keyword=/viaf/search?query=
 _viaf_url=http://www.viaf.org
 
 _conceptpower_authority_search_keyword=ConceptSearch?word=
+_isiscb_authority_search_keyword=authority/search?query=
 
 _upload_files_directory=${upload.folder.path}
 
 _tasks_page_size=20
 _max_export_size=300
 _conceptpower_url=https://chps.asu.edu/conceptpower/rest/
+_isiscb_url=http://isiscb-staging-p3-2.us-west-2.elasticbeanstalk.com/rest/
+
+_isiscb_uri_string=https://data.isiscb.org/isis/authority/{0}/
```

Comment:
this needs to be configurable

Reply:
So, I should move them in properties in pom.xml?